applications/crossbar/priv/couchdb/schemas/callflows.json

```diff
@@ -23,30 +23,8 @@
             "type": "object"
         },
         "flow": {
-            "description": "A callflow node defines a module to execute, data to provide to that module, and one or more children to branch to",
-            "properties": {
-                "children": {
-                    "default": {},
```

Comment:
Turns out the json pretty printer in Emacs incorrectly converts `{"foo":{}}` to `{"foo":null}` when pretty printing. There's a bug filed but it hasn't moved very far afaict. I've changed my pretty-printer to shell out to `python -mjson.tool` instead.